packages/contracts/test-suites/test-aave/eth-call-contracts.spec.ts

```diff
@@ -0,0 +1,15 @@
+import { makeSuite, TestEnv } from "./helpers/make-suite";
+import { GetAllTrancheAssetsData } from "../../types";
+import { deployContract } from "../../helpers/contracts-helpers";
+
+makeSuite("Eth call contracts functionality", (testEnv: TestEnv) => {
+  it("Get all tranche assets data", async () => {
+    const { addressesProvider } = testEnv;
+
+    const c: GetAllTrancheAssetsData =
+      await deployContract<GetAllTrancheAssetsData>("GetAllTrancheAssetsData", [
+        addressesProvider.address,
+        0,
+      ]);
+  });
+});
```

Comment:
should we add the other eth call contracts to this test too

Reply:
no i will remove. we can't actually unit test them since the constructor does not return anything